nengo_spa/examples/intro_coming_from_legacy_spa.ipynb

```diff
@@ -0,0 +1,1183 @@
+    "    ).build()\n",
+    "\n",
+    "    p = nengo.Probe(model.state.output, synapse=0.01)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "with nengo.Simulator(model) as sim:\n",
+    "    sim.run(0.5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.plot(sim.trange(), spa.similarity(sim.data[p], model.vocabs[d]))\n",
+    "plt.xlabel(\"Time [s]\")\n",
+    "plt.ylabel(\"Similarity\")\n",
+    "plt.legend(model.vocabs[d].keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Semantic Pointers\n",
+    "\n",
+    "The `SemanticPointer` class has become immutable in nengo_spa to avoid accidental modifiction of Semantic Pointers. Whenever an operation is applied to a `SemanticPointer` (for example `SemanticPointer(d) + SemanticPointer(d)` or `SemanticPointer(d).normalized()` a new instance will be created.\n",
+    "\n",
+    "To facilitate mathematical analysis of encoding schemes some special vectors are predefined:\n",
+    "\n",
+    "* `nengo_spa.pointer.Identity(d)` gives the identity vector for circular convolution.\n",
+    "* `nengo_spa.pointer.Zero(d)` gives the vector of all zeros (absorbing element for circular convolution).\n",
+    "* `nengo_spa.pointer.AbsorbingElement(d)` gives a vector that destroys all information under circluar convolution except for a DC offset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Representing identity\n",
+    "\n",
+    "In nengo_spa `spa.State()` is optimized to allow the representation of the identity vector. This is not always necessary as in many models the identity never needs to represented. With `represent_identity=False` this optimization can be disabled. This can make the representation for non-identity vectors slightly better. It also simplifies the internal structure of `spa.State()` which can be helpful for applying learning rules."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "## New associative memory classes\n",
+    "\n",
+    "To reduce the number of arguments of the associative memory class it has been split into two classes. `spa.ThresholdingAssocMem` is the applying a simple threshold, `spa.WTAAssocMem` performs a winner-take-all clean-up with lateral inhibitory connections. There is also a new type of associative memory `spa.IAAssocMem` based on independent accumulators, that also exhibits winner-take-all behaviour, but with different dynamics. (TODO link CogSci paper once published)\n",
+    "\n",
+    "To implement auto-associative memories, the `input_keys` and `output_keys` arguments have been replaced by a single `mapping` arguments.\n",
+    "\n",
+    "TODO link associative memory documentation/tutorial."
```

Comment:
It needs updating. Not sure about how to do the linking. Maybe the link can go to the online documentation that should include the examples like in core Nengo.